htsget.md

```diff
@@ -114,14 +114,16 @@ If a request to the URL of an API method includes the `Origin` header, its conte
 The values of `Origin` and `Access-Control-Request-Headers` (if any) of the request will be propagated to `Access-Control-Allow-Origin` and `Access-Control-Allow-Headers` respectively in the preflight response.
 The `Access-Control-Max-Age` of the preflight response is set to the equivalent of 30 days.
 
+# Request
 
-# Method: get reads by ID
+## Methods
 
     GET /reads/<id>
```

Comment:
Should this actually be part of the specs? The EGA API doesn't use '/reads/' at all in the URL, because at the EGA I have to use file IDs; so I am using '/files/' instead. I think this should be more generic, so that different implementations can choose their own way. I do like the addition of '/variants/' for VCF/BCF files.

Reply:
If the URLs are completely arbitrary, how would a client know what endpoint to call to? Yet another one would be necessary to discover which is the URL to search for reads, variants, etc., in a particular server.

Reply:
I'm not sure there's much point in mandating the format of the URL either, since most people are actively ignoring the current 'reads' prefix.  In terms of the client know where to look for reads or variants, we have no idea how a client got the URL in the first place (explicitly out of band), so I think we can assume that the service that provided the URL will know whether it points to reads or variants.